<launch>
  <param name="imu_pub_rate" value="10"/>
  <node pkg="sensor_startup" type="imu_core" name="imu_core" output="screen">
    <param name="port_id" value="/dev/ttyUSB0" />
    <param name="baud_rate" value="115200" />
    <param name="imu_frame_id" value="base_imu"/>
    <param name="imu_pub_topic" value="imu"/>
  </node>
</launch>
